stanford/suclass/ansible.cfg

```diff
@@ -1,9 +1,10 @@
 [defaults]
+callback_whitelist = profile_tasks
 host_key_checking = False
 inventory = ../../playbooks/edx-east/ec2.py
 jinja2_extensions = jinja2.ext.do
 library = ../../playbooks/edx-east/library
-log_path = .ansible.log
+log_path = ../../logs/suclass-ansible.log
 private_key_file = ../../../etc/keys/ssh/suclass/ubuntu.private
 remote_user = ubuntu
-roles_path = ../../playbooks/edx-east/roles
+roles_path = ../../playbooks/roles:../roles
```

Comment:
Is that a valid path?

Reply:
Yes, it's two paths, comma-delimited.

Reply:
We're already using this for stage/lagunita.